features/sign_in_sign_up/forgot_password.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Forgot password

```

Comment:
As I also mentioned in #857, we can rename folder `features/sign_in_sign_up/` into `featuers/sessions/`, it's more broad I think.